### ISSUE TITLE
RedHatOfficial/ansible-role-rhel7-stig fails trying to comment out various properties in sysctl.d files.

### ISSUE THREAD

**Issue by vhaispdeaded** [ComplianceAsCode/content#8275]
Multiple tasks reference entire file attribute object returned by Ansible find task, they should only reference the path attribute.
### Tested Changes (diff of tasks/main.yml)
```
23041c23041
<     path: '{{ item }}'
---
>     path: '{{ item.path }}'
23127c23127
<     path: '{{ item }}'
---
>     path: '{{ item.path }}'
23217c23217
<     path: '{{ item }}'
---
>     path: '{{ item.path }}'
23306c23306
<     path: '{{ item }}'
---
>     path: '{{ item.path }}'
23394c23394
<     path: '{{ item }}'
---
>     path: '{{ item.path }}'
23484c23484
<     path: '{{ item }}'
---
>     path: '{{ item.path }}'
23573c23573
<     path: '{{ item }}'
---
>     path: '{{ item.path }}'
23660c23660
<     path: '{{ item }}'
---
>     path: '{{ item.path }}'
23749c23749
<     path: '{{ item }}'
---
>     path: '{{ item.path }}'
23842c23842
<     path: '{{ item }}'
---
>     path: '{{ item.path }}'
23934c23934
<     path: '{{ item }}'
---
>     path: '{{ item.path }}'
24894c24894
<     path: '{{ item }}'
---
>     path: '{{ item.path }}'
```

## Actual Results (example task)
```
TASK [RedHatOfficial.rhel7_stig : Comment out any occurrences of net.ipv6.conf.all.accept_source_route from /etc/sysctl.d/*.conf files] ************************************************************************************************************************
failed: [ip-10-247-148-27.us-gov-west-1.compute.internal] (item={u'rusr': True, u'uid': 0, u'rgrp': True, u'xoth': False, u'islnk': False, u'woth': False, u'nlink': 1, u'issock': False, u'mtime': 1628783374.0, u'gr_name': u'root', u'path': u'/etc/sysctl.d/98-stig.conf', u'xusr': False, u'atime': 1645654258.0, u'inode': 322991, u'isgid': False, u'size': 310, u'isdir': False, u'ctime': 1645656724.488381, u'isblk': False, u'wgrp': False, u'xgrp': False, u'isuid': False, u'dev': 64768, u'roth': True, u'isreg': True, u'isfifo': False, u'mode': u'0644', u'pw_name': u'root', u'gid': 0, u'ischr': False, u'wusr': True}) => {"ansible_loop_var": "item", "changed": false, "item": {"atime": 1645654258.0, "ctime": 1645656724.488381, "dev": 64768, "gid": 0, "gr_name": "root", "inode": 322991, "isblk": false, "ischr": false, "isdir": false, "isfifo": false, "isgid": false, "islnk": false, "isreg": true, "issock": false, "isuid": false, "mode": "0644", "mtime": 1628783374.0, "nlink": 1, "path": "/etc/sysctl.d/98-stig.conf", "pw_name": "root", "rgrp": true, "roth": true, "rusr": true, "size": 310, "uid": 0, "wgrp": false, "woth": false, "wusr": true, "xgrp": false, "xoth": false, "xusr": false}, "msg": "Path {'uid': 0, 'woth': False, 'mtime': 1628783374.0, 'inode': 322991, 'isgid': False, 'size': 310, 'roth': True, 'isuid': False, 'isreg': True, 'pw_name': 'root', 'gid': 0, 'ischr': False, 'wusr': True, 'xoth': False, 'islnk': False, 'nlink': 1, 'issock': False, 'rgrp': True, 'gr_name': 'root', 'path': '/etc/sysctl.d/98-stig.conf', 'xusr': False, 'atime': 1645654258.0, 'isdir': False, 'ctime': 1645656724.488381, 'isblk': False, 'xgrp': False, 'dev': 64768, 'wgrp': False, 'isfifo': False, 'mode': '0644', 'rusr': True} does not exist !", "rc": 257}

PLAY RECAP *****************************************************************************************************************************************************************************************************************************************************
ip-10-247-148-27.us-gov-west-1.compute.internal : ok=1608 changed=207  unreachable=0    failed=1    skipped=527  rescued=0    ignored=0   
```
## Expected Results (example task)
```
TASK [RedHatOfficial.rhel7_stig : Comment out any occurrences of net.ipv6.conf.all.accept_source_route from /etc/sysctl.d/*.conf files] ***************************************************************************************************************************
changed: [ip-10-247-148-27.us-gov-west-1.compute.internal] => (item={u'rusr': True, u'uid': 0, u'rgrp': True, u'xoth': False, u'islnk': False, u'woth': False, u'nlink': 1, u'issock': False, u'mtime': 1645824666.158254, u'gr_name': u'root', u'path': u'/etc/sysctl.d/98-stig.conf', u'xusr': False, u'atime': 1645824666.158254, u'inode': 4913038, u'isgid': False, u'size': 311, u'isdir': False, u'ctime': 1645824666.160254, u'isblk': False, u'wgrp': False, u'xgrp': False, u'isuid': False, u'dev': 64768, u'roth': True, u'isreg': True, u'isfifo': False, u'mode': u'0644', u'pw_name': u'root', u'gid': 0, u'ischr': False, u'wusr': True})
```

**Comment by ggbecker** [comment]
Fixed by #8462

**Comment by vhaispdeaded** [comment]
Thank you
